python/lsst/pipe/tasks/postprocess.py

```diff
@@ -1299,3 +1300,75 @@ def writeConfig(self, butler, clobber=False, doBackup=True):
         """No config to write.
         """
         pass
+
+
+class MakeVisitTableConnections(pipeBase.PipelineTaskConnections,
+                                dimensions=("instrument",),
+                                defaultTemplates={}):
+    visitSummaries = pipeBase.connectionTypes.Input(
+        doc="Per-visit consolidated exposure metadata from ConsolidateVisitSummaryTask",
+        name="visitSummary",
+        storageClass="ExposureCatalog",
+        dimensions=("instrument", "visit",),
+        multiple=True,
+        deferLoad=True,
+    )
+    outputCatalog = connectionTypes.Output(
+        doc="Visit metadata table",
+        name="visitTable",
+        storageClass="DataFrame",
+        dimensions=("instrument",)
+    )
+
+
+class MakeVisitTableConfig(pipeBase.PipelineTaskConfig,
+                           pipelineConnections=MakeVisitTableConnections):
+    pass
+
+
+class MakeVisitTableTask(CmdLineTask, pipeBase.PipelineTask):
+    """Produce a `visitTable` from the `visitSummary` exposure catalogs.
+    """
+    _DefaultName = 'makeVisitTable'
+    ConfigClass = MakeVisitTableConfig
+
+    def run(self, visitSummaries):
+        """ Make a table of visit information from the `visitSummary` catalogs
+
+        Parameters
+        ----------
+        visitSummaries : list of `lsst.afw.table.ExposureCatalog`
+            List of exposure catalogs with per-detector summary information.
+
+        Returns
+        -------
+        result : `lsst.pipe.Base.Struct`
+            Results struct with attribute:
+                ``outputCatalog``
+                    Catalog of visit information.
+        """
+        visitEntries = []
+        for visitSummary in visitSummaries:
+            visitSummary = visitSummary.get()
+
+            visitRow = visitSummary[0]
+            visitInfo = visitRow.getVisitInfo()
+            visitEntry = {}
+            visitEntry["visitId"] = visitRow['visit']
+            visitEntry["filterName"] = visitRow['physical_filter']
+            visitEntry["ra"] = np.mean(visitSummary['ra'])
+            visitEntry["decl"] = np.mean(visitSummary['decl'])
+            visitEntry["skyRotation"] = visitInfo.getBoresightRotAngle().asRadians()
+            azimuth, altitude = visitInfo.getBoresightAzAlt()
+            visitEntry["azimuth"] = azimuth.asRadians()
+            visitEntry["altitude"] = altitude.asRadians()
```

Comment:
All units in this table according to the baseline schema should be degrees, e.g. https://github.com/lsst/sdm_schemas/blob/master/yml/baselineSchema.yaml#L5527